sass/etna.scss

```diff
@@ -236,12 +237,15 @@ $largest-small-device: 992px;
     }
 }
 
+/* stylelint-disable no-descending-specificity */
```

Comment:
I've used disables where I don't want to risk any changes that might affect the styling result